### PR TITLE
Removing stale ref to UW resource

### PR DESCRIPTION
**Pull request by joemcl**
Removing invalid link to U Wisconsin resource - no longer on that page.

### CODE REVIEW

File: docs/tools-reference/linux-users-and-groups/index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ modified_by:
 published: 2009-08-31
 title: Linux Users and Groups
 external_resources:
- - '[User Account and Group Management @ UWISC''s Center for Computer Aided Engineering](http://www.cae.wisc.edu/linaccounts)'
```
Comment (bagasme) [inline docs/tools-reference/linux-users-and-groups/index.md]: Broken link?

Comment (joemcl) [reply docs/tools-reference/linux-users-and-groups/index.md]: @bagasme Yes but can't find the content on the page anywhere else, even

Comment (bagasme) [reply docs/tools-reference/linux-users-and-groups/index.md]: After reviewing [web.archive.org snapshot](http://web.archive.org/web/20160926055501/http://www.cae.wisc.edu/cae-account-management/), actually it is account management home page for CAE course, not related to our Linux Users & Groups guide.

So removing this external resource is correct, to keep links related.

Comment (bagasme) [reply docs/tools-reference/linux-users-and-groups/index.md]: Indeed, the external link above bounce to course account management page.

Comment (bagasme) [reply docs/tools-reference/linux-users-and-groups/index.md]: Edit: It is actually broken link without redirection

Comment (joemcl) [reply docs/tools-reference/linux-users-and-groups/index.md]: right

**Pull request merged**